### PR TITLE
Upgrade dependencies and build settings

### PR DESCRIPTION
**Pull request by maxime-rainville**
Upgrade dependencies and webpack set up

# Parent issue
* https://github.com/silverstripeltd/product-issues/issues/48

### CODE REVIEW

File: templates/CommentsInterface.ss [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<% require themedCSS('comments', 'comments') %>
+<% require themedCSS('client/dist/styles/comments', 'comments') %>
```
Comment (maxime-rainville) [inline templates/CommentsInterface.ss:1]: The second parameter is meant to be a media  attribute. I'm pretty sure `comments` is a nonsensical value here. However, if I remove it, there's a good chance we'll end up injecting CSS in people front end without them noticing.

**Comment by maxime-rainville** [comment]
This moves some CSS and JS files around. This may be problematic for some people if they've manually reference those files.

I would like a second opinion on whatever this is kosher or not.

**Comment by bergice** [comment]
> This moves some CSS and JS files around. This may be problematic for some people if they've manually reference those files.
> 
> I would like a second opinion on whatever this is kosher or not.

Maybe this should target the next major version in that case?

**Comment by maxime-rainville** [comment]
I had another look at this thing. The only files that is loaded in loaded via a template, is the `css/comments.css` file. This file is literally a big comment, so it doesn't do anything. Even if it did anything, there's a bad condition on the `themedCSS` call to limit the CSS to the 'comments' media, which is a non-sensical value.

The current file imports are broken any way, so we might as well have the new build set up and get it over with.

**Pull request merged**